Gemfile.lock

```diff
@@ -35,4 +35,4 @@ DEPENDENCIES
   rspec
 
 BUNDLED WITH
-   1.17.1
```

Comment:
Are the changes of the .lock necessary?

Reply:
@bronzdoc I don't think these changes are necessary. I just removed that commit. 👍